README.md

```diff
@@ -24,6 +24,8 @@ pip install jupyterlab_myst
 
 ## Usage
 
+[MyST}] is a flavour of Markdown, which combines the fluid experience of writing Markdown with the programmable extensibility of reStructuredText. This extension for JupyterLab makes it easier to develop rich, computational narratives, technical documentation, and open scientific communication.
```

Comment:
Is the punctuation intentional? If we rebase, we can make this a link easily with your other changes.

```suggestion
MyST is a flavour of Markdown, which combines the fluid experience of writing Markdown with the programmable extensibility of reStructuredText. This extension for JupyterLab makes it easier to develop rich, computational narratives, technical documentation, and open scientific communication.
```

Reply:
A very generous way of saying wtf is this ;) It was a link at one point, but something in my text buffer seems to have been moved around.

Reply:
```suggestion
[MyST](https://myst-tools.org/docs/mystjs/quickstart-myst-markdown) is a flavour of Markdown, which combines the fluid experience of writing Markdown with the programmable extensibility of reStructuredText. This extension for JupyterLab makes it easier to develop rich, computational narratives, technical documentation, and open scientific communication.
```

Reply:
Was wondering if it was a part of the logo or something!